lib/bigtable/index.js

```diff
@@ -0,0 +1,475 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module bigtable
+ */
+
+'use strict';
+
+var googleProtoFiles = require('google-proto-files');
+var is = require('is');
+var nodeutil = require('util');
+var format = require('string-format-obj');
+
+/**
+ * @type {module:bigtable/table}
+ * @private
+ */
+var Table = require('./table.js');
+
+/**
+ * @type {module:common/grpcService}
+ * @private
+ */
+var GrpcService = require('../common/grpc-service.js');
+
+/**
+ * @type {module:common/util}
+ * @private
+ */
+var util = require('../common/util.js');
+
+/**
+ * Interact with
+ * [Google Cloud Bigtable](https://cloud.google.com/bigtable/).
+ *
+ * @constructor
+ * @alias module:bigtable
+ *
+ * @classdesc
+ * The `gcloud.bigtable` object allows you interact with Google Cloud Bigtable.
+ *
+ * To learn more about Bigtable, read the
+ * [Google Cloud Bigtable Concepts Overview](https://cloud.google.com/bigtable/docs/concepts)
+ *
+ * @resource [Creating a Cloud Bigtable Cluster]{@link https://cloud.google.com/bigtable/docs/creating-cluster}
+ *
+ * @param {object=} options - [Configuration object](#/docs).
+ * @param {string} options.cluster - The cluster name that hosts your tables.
+ * @param {string|module:compute/zone} options.zone - The zone in which your
+ *     cluster resides.
+ *
+ * @example
+ * var gcloud = require('gcloud')({
+ *   keyFilename: '/path/to/keyfile.json',
+ *   projectId: 'my-project'
+ * });
+ *
+ * var bigtable = gcloud.bigtable({
```

Comment:
A common scenario is for users to have a Bigtable cluster in one project and share that cluster with other projects.  So, we typically think of project vs. cluster-project.  This is more explicit in v2.